### PR TITLE
Add support to build multi-platform images

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/DockerfileBuilder.java [hunk 1]
```diff
@@ -57,7 +57,24 @@ public void build() throws IOException, InterruptedException {
...
+        if (dockerSettings.isBuildx()) {
+            String output;
+            switch (dockerSettings.getOutput()) {
+                case "push":
+                    output = "--push";
+                    break;
+                case "load":
+                    output = "--load";
+                    break;
+                default:
```
Comment (oleg-nenashev) [inline custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/DockerfileBuilder.java:69]: Fail on null and default?

**Review by oleg-nenashev — APPROVED** [review]
Looks good, thanks!
A demo or a test would be useful, but the logic is quite straightforward.

File: custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/DockerfileBuilder.java (hunk 1 above)
Comment (LinuxSuRen) [reply custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/DockerfileBuilder.java:71]: I add some comments in the code lines. The default behaviour is correct.

**Review by oleg-nenashev — APPROVED** [review]
Thank you!

**Comment by LinuxSuRen** [comment]
hi @oleg-nenashev , I'm wondering if it's possible to create a new release for this project. For example: `v2.0-alpha-6`. I know there are not too many features or bug fixes between `v2.0-alpha-5` and [the latest branch](https://github.com/jenkinsci/custom-war-packager/tree/e4d8fca52eb526876364e3a9adcfcff964bdbbeb). But it already passed for serval months.

**Comment by oleg-nenashev** [comment]
Will do

**Comment by LinuxSuRen** [comment]
> Will do

Thanks for your quick response.

**Pull request merged**